go/protocol/keybase1/extras.go

```diff
@@ -1269,7 +1269,7 @@ func (k TeamApplicationKey) Generation() int {
 	return k.KeyGeneration
 }
 
-func (t TeamMembers) All() []string {
+func (t TeamMembers) AllUsernames() []string {
```

Comment:
Sorry I didn't mention this earlier. Should these be `libkb.NormalizedUsername`s?